flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```

Comment:
Yeah, they let flow type check code which uses those libraries. I should have merged it earlier, but I wasn't using flow properly.